### ISSUE TITLE
AVT 1 - Tile Multi-Select: DAP violations 

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon-components-react#2238]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Detailed Description
Run DAP with Feb. 2019 ruleset on the Tile Multi-select component
See screenshot below for violations related to id's and labels that are not unique:
<img width="1204" alt="Screen Shot 2019-04-18 at 11 22 55 AM" src="https://user-images.githubusercontent.com/21676914/56376095-ab7f9a80-61cc-11e9-84b0-4b776b0abeb6.png">
